pkg/client/group.go

```diff
@@ -102,13 +102,60 @@ func (c *Client) GetGroupPage(ctx context.Context, groupID int64, pageID int64)
 	return &ret, nil
 }
 
+func (c *Client) InsertGroupPage(ctx context.Context, groupID int64, pageID int64, accessLevel string) error {
+	l := ctxzap.Extract(ctx)
+	l.Debug("inserting group page", zap.Int64("group_id", groupID), zap.Int64("page_id", pageID), zap.String("access_level", accessLevel))
+
+	args := []interface{}{groupID, pageID, accessLevel}
+	sb := &strings.Builder{}
+	_, _ = sb.WriteString(`INSERT INTO group_pages ("groupId", "pageId", "accessLevel", "createdAt", "updatedAt") VALUES ($1, $2, $3, NOW(), NOW())`)
+
+	if _, err := c.db.Exec(ctx, sb.String(), args...); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (c *Client) UpdateGroupPage(ctx context.Context, id int64, accessLevel string) error {
+	l := ctxzap.Extract(ctx)
+	l.Debug("updating group page", zap.Int64("id", id), zap.String("access_level", accessLevel))
+
+	args := []interface{}{accessLevel, id}
+	sb := &strings.Builder{}
+	_, _ = sb.WriteString(`UPDATE group_pages SET "accessLevel" = $1 WHERE "id"=$2`)
+
+	_, err := c.db.Exec(ctx, sb.String(), args...)
+
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (c *Client) DeleteGroupPage(ctx context.Context, id int64) error {
+	l := ctxzap.Extract(ctx)
+	l.Debug("deleting group page", zap.Int64("id", id))
+
+	args := []interface{}{id}
+	sb := &strings.Builder{}
+	_, _ = sb.WriteString(`DELETE FROM group_pages WHERE "id"=$1`)
+
+	if _, err := c.db.Exec(ctx, sb.String(), args...); err != nil {
+		return err
+	}
+
+	return nil
+}
+
 func (c *Client) GetGroupFolderDefault(ctx context.Context, groupID int64, folderID int64) (*GroupFolderDefault, error) {
 	l := ctxzap.Extract(ctx)
 	l.Debug("getting group folder default", zap.Int64("group_id", groupID), zap.Int64("folder_id", folderID))
 
 	args := []interface{}{groupID, folderID}
 	sb := &strings.Builder{}
-	_, _ = sb.WriteString(`select "id", "accessLevel" from group_folder_defaults WHERE "groupId"=$1 AND "folderId"=$2`)
+	_, _ = sb.WriteString(``)
```

Comment:
should that remain without a SQL statement tho?

Reply:
It was not supposed to be empty, I fix it.